apps/system/test/marionette/browser_video_test.js

```diff
@@ -76,6 +76,7 @@ marionette('Video', function() {
       return player.isFullScreen();
     });
 
+    client.helper.wait (1000);
```

Comment:
The control panel shows every now and then a strange behavior in that very moment the test switches back from full-screen. It reappears for a very short period of time. In essence this wait-call led to 50+ passing tests in sequence.